rasa/core/actions/forms.py

```diff
@@ -463,7 +460,8 @@ async def request_next_slot(
                 return [SlotSet(REQUESTED_SLOT, slot), *bot_message_events]
 
         # no more required slots to fill
-        return None
+        # TODO: Is it ok if we change the order of setting this slot to None and Form(None)?
```

Comment:
what do you mean? first slot to none then form to none? should be fine, actually since it happens inside 1 dialogue turn, I don't think the order matters

Reply:
awesome, will remove the todos